### ISSUE TITLE
Potential issue in equation (11) of paper

### ISSUE THREAD

**Issue by pwein-liveeo** [likyoo/SemiCD-VL#3]
Dear authors,

Thank you for the excellent paper and for sharing your code!

I believe there might be a small issue in equation (11) of the [paper](https://arxiv.org/abs/2405.04788v5). From my understanding, the conditions appear to be reversed. The value 255 should be selected when `I_rel(k) == 0`. The stated condition in v5 is `I_rel(k) == 1`. Similarly, for the first row, the condition should be `I_rel(k) == 1`. This ensures that change maps are only created where the VLM is highly confident in the semantic segmentations of both timesteps, and locations with low confidence for at least one timestep are ignored (255).

This seems consistent with the code in the [script](https://github.com/likyoo/SemiCD-VL/blob/main/scripts/gen_cd_map.py).

Could you please confirm whether there is an error in the paper or if I have misunderstood?

Thank you!

**Comment by likyoo** [comment]
You're right! Thank you for pointing this out, we will correct it in a subsequent version. 😄

**Comment by pwein-liveeo** [comment]
Thanks for the extremely fast response
